helpers/deploymentConfig.ts

```diff
@@ -147,6 +147,7 @@ export enum InterestRateModels {
 
 const ANY_CONTRACT = ethers.constants.AddressZero;
 
+export const DEFAULT_BLOCKS_PER_YEAR = 10_512_000; // assuming a block is mined every 3 seconds
```

Comment:
Let's set `21_024_000`, one block every 1.5 seconds, to check the effects of the next upgrade on BNB Chain